File: carrera-sdk/producer/go/src/carrera/cancelTxMonitorMessageBuilder.go
package carrera

import "carrera/CarreraProducer"

type CancelTxMonitorMessageBuilder struct {
	cancelDelayMessageBuilder *CancelDelayMessageBuilder
}

func NewCancelTxMonitorMessageBuilder(cancelDelayMessageBuilder *CancelDelayMessageBuilder) *CancelTxMonitorMessageBuilder {
	return &CancelTxMonitorMessageBuilder{cancelDelayMessageBuilder:cancelDelayMessageBuilder}
}

func (cancelTxMonitorMessageBuilder *CancelTxMonitorMessageBuilder) Send() *CarreraProducer.DelayResult {
	return cancelTxMonitorMessageBuilder.cancelDelayMessageBuilder.Send()
}

File: carrera-sdk/producer/java/carrera-producer-sdk/src/main/java/com/xiaojukeji/carrera/producer/tx/CancelTxMonitorMessageBuilder.java
package com.xiaojukeji.carrera.producer.tx;

import com.xiaojukeji.carrera.producer.CancelDelayMessageBuilder;
import com.xiaojukeji.carrera.thrift.DelayResult;


public class CancelTxMonitorMessageBuilder {

    private CancelDelayMessageBuilder cancelDelayMessageBuilder;

    public CancelTxMonitorMessageBuilder(CancelDelayMessageBuilder cancelDelayMessageBuilder) {
        this.cancelDelayMessageBuilder = cancelDelayMessageBuilder;
    }

    public DelayResult send() {
        return this.cancelDelayMessageBuilder.send();
    }
}